vars/packageBuildingPipelineDAOSTest.groovy

```diff
@@ -809,7 +810,9 @@ void call(Map pipeline_args) {
                                                                   !('leap15' in distros)),
                                                    booleanParam(name: 'CI_DEB_Ubuntu20_NOBUILD',
                                                                 value: pipeline_args.get('skip-build', true) ||
-                                                                       !('ubuntu20.04' in distros))
+                                                                       !('ubuntu20.04' in distros)),
+                                                   booleanParam(name: 'CI_ALLOW_UNSTABLE_TEST',
+                                                                value: cachedCommitPragma('Allow-unstable-test').toLowerCase() == 'true')
```

Comment:
Should this be getting the `Allow-unstable-test` argument (commit pragma) from the `pipeline_args` map - similar to `skip-build`, e.g.?
```suggestion
                                                                value: pipeline_args.get('Allow-unstable-test', false)
```